packages/elements/src/select/index.ts

```diff
@@ -53,6 +45,13 @@ const LABEL_SEPARATOR = ', '; // TODO: for multiselect
 const POPUP_POSITION = ['bottom-start', 'top-start'];
 const KEY_SEARCH_DEBOUNCER = 300;
 
+enum Direction {
```

Comment:
`Direction` essentially means moving towards something. `First`, `Last`, `Next` and `Previous` is not a direction, but destination :) so I may suggest to rename it to `enum Navigation`

Reply:
Agree, navigation sounds better.